tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py

```diff
@@ -89,7 +90,38 @@ def test_onboarding_data_assistant_result_serialization(
         bobby_onboarding_data_assistant_result.to_json_dict()
         == onboarding_data_assistant_result_as_dict
     )
-    assert len(bobby_onboarding_data_assistant_result.profiler_config.rules) == 8
+    assert len(bobby_onboarding_data_assistant_result.profiler_config.rules) == 7
+
+
+def test_onboarding_data_assistant_metrics_count(
+    bobby_onboarding_data_assistant_result: OnboardingDataAssistantResult,
+) -> None:
+    domain: Domain
+    parameter_values_for_fully_qualified_parameter_names: Dict[str, ParameterNode]
+    num_metrics: int
+
+    domain_key: Domain = Domain(
+        domain_type=MetricDomainTypes.TABLE,
+    )
+
+    num_metrics = 0
+    for (
+        domain,
+        parameter_values_for_fully_qualified_parameter_names,
+    ) in bobby_onboarding_data_assistant_result.metrics_by_domain.items():
+        if domain.is_superset(domain_key):
+            num_metrics += len(parameter_values_for_fully_qualified_parameter_names)
+
+    assert num_metrics == 2
+
+    num_metrics = 0
+    for (
+        domain,
+        parameter_values_for_fully_qualified_parameter_names,
+    ) in bobby_onboarding_data_assistant_result.metrics_by_domain.items():
+        num_metrics += len(parameter_values_for_fully_qualified_parameter_names)
+
+    assert num_metrics == 178
```

Comment:
Can we also assert that `num_metrics >= len(bobby_onboarding_data_assistant_result.expectation_configurations)`?